CI.hs

```diff
@@ -122,3 +122,12 @@ main = do
         writeFile file .
           replace "ghc-lib-parser" ("ghc-lib-parser == " ++ version)
           =<< readFile' file
+
+      removePath :: FilePath -> IO ()
+      removePath p = do
+        whenM (doesFileExist p) $ do
```

Comment:
Does the distinction between files and directories matter all that much? I would just use `doesPathExist` and only have a single case.